### PR TITLE
Fix issue where xml mapping cannot find file on certain configurations

### PR DESCRIPTION
**Pull request by mistraal**
Please refer to [this issue](https://github.com/wallabag/wallabag/issues/1828) and [this issue](https://github.com/FriendsOfSymfony/FOSUserBundle/issues/1062). 

While this is a very specific case, it is not exclusive to Nginx server as it also happens on my Apache setup.

### CODE REVIEW

**Comment by Ocramius** [comment]
This needs a test case.

File: lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php [hunk 1]
```diff
@@ -911,7 +911,8 @@ private function getCascadeMappings(SimpleXMLElement $cascadeElement)
     protected function loadMappingFile($file)
     {
         $result = [];
-        $xmlElement = simplexml_load_file($file);
+        $xmlData = file_get_contents($file);
```
Comment (Ocramius) [inline lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php]: Can be inlined

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php [hunk 2]
```diff
@@ -911,7 +911,7 @@ private function getCascadeMappings(SimpleXMLElement $cascadeElement)
     protected function loadMappingFile($file)
     {
         $result = [];
-        $xmlElement = simplexml_load_file($file);
+        $xmlElement = simplexml_load_string(file_get_contents($file));
```
Comment (phansys) [inline lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php]: Is there any situation where [`simplexml_load_file()`](http://php.net/manual/en/function.simplexml-load-file.php) is not returning `false` with a not found file?

**Review by phansys — CHANGES_REQUESTED** [review]
Status: Answer pending.

File: lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php (hunk 2 above)
Comment (Ocramius) [inline lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php]: > Is there any situation where [`simplexml_load_file()`](http://php.net/manual/en/function.simplexml-load-file.php) is not returning `false` with a not found file?

Indeed, that's what a test would verify, but I see no tests.

**Comment by phansys** [comment]
This is related to #930.

**Comment by lcobucci** [comment]
@mistraal can you please rebase with `develop` dropping all commits that aren't yours please?

**Pull request closed without merging**